storage/persistent_storage/persistent-storage-gce.adoc

```diff
@@ -25,14 +25,14 @@ requested by users.
 
 [IMPORTANT]
 ====
-{product-title} defaults to using an in-tree (non-CSI) plugin to provision gcePD storage.
+{product-title} 4.12 and later provides automatic migration for GCE Persist Disk in-tree volume plugin to its equivalent CSI driver.
 
-In future {product-title} versions, volumes provisioned using existing in-tree plug-ins are planned for migration to their equivalent CSI driver. CSI automatic migration should be seamless. Migration does not change how you use all existing API objects, such as persistent volumes, persistent volume claims, and storage classes.
+CSI automatic migration should be seamless. Migration does not change how you use all existing API objects, such as persistent volumes, persistent volume claims, and storage classes.
 ifndef::openshift-dedicated[]
 For more information about migration, see xref:../../storage/container_storage_interface/persistent-storage-csi-migration.adoc#persistent-storage-csi-migration[CSI automatic migration].
 endif::openshift-dedicated[]
 
-After full migration, in-tree plugins will eventually be removed in future versions of {product-title}.
+In-tree plugins will eventually be removed in future versions of {product-title}.
```

Comment:
```suggestion
In-tree plugins are expected to be removed in future versions of {product-title}.
```
See https://redhat-documentation.github.io/supplementary-style-guide/#statements-about-the-future